index.js

```diff
@@ -42,11 +42,18 @@ function loadRcConfig(callback){
 				var options;
 
 				if(!err) {
+					if (file === "") {
+						file = "{}";
+					}				
```

Comment:
Let's leave the existing functionality as it is, so no need to set the file variable